models/unit.go

```diff
@@ -9,13 +9,14 @@ type UnitType int
 
 // Enumerate all the unit types
 const (
-	UnitTypeCode            UnitType = iota + 1 // 1 code
-	UnitTypeIssues                              // 2 issues
-	UnitTypePullRequests                        // 3 PRs
-	UnitTypeReleases                            // 4 Releases
-	UnitTypeWiki                                // 5 Wiki
-	UnitTypeExternalWiki                        // 6 ExternalWiki
-	UnitTypeExternalTracker                     // 7 ExternalTracker
+	UnitTypeCode              UnitType = iota + 1 // 1 code
+	UnitTypeIssues                                // 2 issues
+	UnitTypePullRequests                          // 3 PRs
+	UnitTypeReleases                              // 4 Releases
+	UnitTypeWiki                                  // 5 Wiki
+	UnitTypeExternalWiki                          // 6 ExternalWiki
+	UnitTypeExternalTracker                       // 7 ExternalTracker
+	UnitTypeIssueDependencies                     // 8 Issue Dependencies
```

Comment:
I don't see point of creating `UnitType` for this. It is same as time tracker.

Reply:
I didn't know that, I just copied what was there and adapted it...

But I will make it like the time trackingsettings.

Reply:
@Morlinest Done.